cmd/fuego/templates/newEntity/controller.go

```diff
@@ -0,0 +1,65 @@
+package newEntity
+
+import (
+	"github.com/go-fuego/fuego"
+)
+
+type NewEntityResources struct {
+	// TODO add resources
+	NewEntityService NewEntityService
+}
+
+func (rs NewEntityResources) Routes(s *fuego.Server) {
+	newEntityGroup := fuego.Group(s, "/newEntity")
+
+	fuego.Get(newEntityGroup, "/", rs.getAllNewEntity)
+	fuego.Post(newEntityGroup, "/", rs.postNewEntity)
+
+	fuego.Get(newEntityGroup, "/{id}", rs.getNewEntity)
+	fuego.Put(newEntityGroup, "/{id}", rs.putNewEntity)
+	fuego.Delete(newEntityGroup, "/{id}", rs.deleteNewEntity)
+}
+
+func (rs NewEntityResources) getAllNewEntity(c fuego.ContextNoBody) ([]NewEntity, error) {
+	return rs.NewEntityService.GetAllNewEntity()
+}
+
+func (rs NewEntityResources) postNewEntity(c *fuego.ContextWithBody[NewEntityCreate]) (NewEntity, error) {
+	body, err := c.Body()
+	if err != nil {
+		return NewEntity{}, err
+	}
+
+	new, err := rs.NewEntityService.CreateNewEntity(body)
+	if err != nil {
+		return NewEntity{}, err
+	}
+
+	return new, nil
+}
+
+func (rs NewEntityResources) getNewEntity(c fuego.ContextNoBody) (NewEntity, error) {
+	id := c.PathParam("id")
+
+	return rs.NewEntityService.GetNewEntity(id)
+}
+
+func (rs NewEntityResources) putNewEntity(c *fuego.ContextWithBody[NewEntityUpdate]) (NewEntity, error) {
+	id := c.PathParam("id")
+
+	body, err := c.Body()
+	if err != nil {
+		return NewEntity{}, err
+	}
+
+	new, err := rs.NewEntityService.UpdateNewEntity(id, body)
```

Comment:
is this generated?

Reply:
What do you mean?

Reply:
I'm asking if it was generated from like codegen or something 

For instance this can be `return rs.NewEntityService.UpdateNewEntity(id, body)`

Reply:
This will be replaced by `return rs.BookService.UpdateBook(id, body)`

Reply:
This file is just a template.